test/unit/specs/store/wallet.spec.js

```diff
@@ -245,4 +262,68 @@ describe(`Module: Wallet`, () => {
     })
     expect(state.error.message).toBe(`Error`)
   })
+
+  it(`should send coins`, async () => {
+    state.balances = [
+      {
+        denom: `funcoin`,
+        amount: 1000
+      }
+    ]
+
+    const commit = jest.fn()
```

Comment:
`commit` seems not used, maybe we can merge the two expectations in a single `it`?